### PR TITLE
Add return type to `getKlines`

### PR DESCRIPTION
**Pull request by mkosir**
## Summary
PR adds (reuses from futures) `Kline` type to spot `getKlines` function.

### CODE REVIEW

File: src/types/shared.ts [hunk 1]
```diff
@@ -141,6 +141,22 @@ export interface KlinesParams {
...
+  numberInString, // close
+  numberInString, // volume
+  number, // close time
+  numberInString, // quote asset volume
+  number, // number of trades
+  numberInString, // taker buy base asset vol
+  numberInString, // taker buy quote asset vol
+  numberInString // ignore?
+];
+
```
Comment (tiagosiebler) [inline src/types/shared.ts:159]:
```suggestion
export type Kline = [
  number, // open time
  numberInString, // open
  numberInString, // high
  numberInString, // low
  numberInString, // close
  numberInString, // volume
  number, // close time
  numberInString, // quote asset volume
  number, // number of trades
  numberInString, // taker buy base asset vol
  numberInString, // taker buy quote asset vol
  numberInString // ignore?
];

/** @deprecated `FuturesKline` will be removed soon. Use `Kline` instead. **/
export type FuturesKline = Kline;
```

This is great, thank you! Tiny nitpick, could you add a backwards compatible type like this preserving the old name?

By removing the old one you may be causing a breaking change, and I'd rather avoid those for minor patch releases.

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
This is great thank you. Tiny nitpick around renaming an existing exported type but otherwise great.

**Review by tiagosiebler — APPROVED** [review]
Actually, that was an easy commit via the suggested change. No action needed, will merge shortly. Thanks!

**Comment by mkosir** [comment]
> This is great thank you. Tiny nitpick around renaming an existing exported type but otherwise great.

Thank you for quick resolve in merge on this PR 🚀

**Pull request merged**